tool-populate/api/Roles.js

```diff
@@ -0,0 +1,51 @@
+var _ = require('lodash')
+var Parse = require('parse/node')
+var Store = require('../data/Store')
+
+function loadRoles(options, callback) {
+  var Role = Parse.Role
+  var query = new Parse.Query(Role)
+  query.limit = 1000
+
+  query.find({
+    success: function(results) {
+      storeRoles(results)
+      if (callback)
+        callback(null, results)
+    },
+    error: function(error, results) {
+      console.warn("Error: " + error.code + " " + error.message)
+      if (callback)
+        callback(error, results)
+    }
+  })
+}
+
+function createRoles(parseRoles) {
+  if (parseRoles.length)
+    return;
+
+  var roleACL = new Parse.ACL();
+  roleACL.setPublicReadAccess(true);
+  var role = new Parse.Role("admin", roleACL);
+  role.save(null, {
+    success: function(response) {
+      storeRoles(response)
```

Comment:
Looking over this code, I see a lot of in-memory mapping of these remote collections to Store.js. So removing Store.js from this script will be a significant chore.  Let me know.